lib/trino-parquet/src/test/java/io/trino/parquet/TestTupleDomainParquetPredicate.java

```diff
@@ -842,4 +891,47 @@ private static List<ByteBuffer> toByteBufferList(Long... values)
         }
         return buffers;
     }
+
+    private static class MockBloomFilterStore
+            extends BloomFilterStore
+    {
+        final Map<ColumnPath, BloomFilter> bloomFilterMap;
+
+        MockBloomFilterStore(Map<ColumnPath, BloomFilter> bloomFilterMap)
+        {
+            super(new MockParquetDataSource(), new BlockMetaData(), new HashSet<>());
+            this.bloomFilterMap = requireNonNull(bloomFilterMap);
+        }
+
+        @Override
+        public Optional<BloomFilter> readBloomFilter(ColumnPath columnPath)
+        {
+            return Optional.ofNullable(bloomFilterMap.getOrDefault(columnPath, null));
```

Comment:
bloomFilterStore.getOrDefault(columnPath, null) -> bloomFilterStore.get(columnPath)

Reply:
it is bloomFilterMap not bloomFilterStore, fixed, since"
the value to which the specified key is mapped, or null if this map contains no mapping for the key"